lib/krane/deploy_task.rb

```diff
@@ -74,7 +74,7 @@ def predeploy_sequence
         Pod
       ).map { |r| [r, default_group] }
 
-      crs = cluster_resource_discoverer.crds.select(&:predeployed?).map { |cr| [cr.kind, { group: cr.group }] }
+      crs = cluster_resource_discoverer.crds.select(&:predeployed?).map { |cr| [cr.group_kind, { group: cr.group }] }
```

Comment:
the redundancy of duplicating group information makes this fix much easier to implement, and is really just an internal consideration, anyway, so it felt like it was worth it.